tests/unit/Codeception/Module/TestsForWeb.php

```diff
@@ -691,6 +691,17 @@ public function testGrabValueFrom()
         $this->module->amOnPage('/form/select');
         $result = $this->module->grabValueFrom('#age');
         $this->assertEquals('oldfag', $result);
+
+        $this->module->amOnPage('/form/bug3866');
```

Comment:
big thanks for this PR, and especially for test, but could you move this to separate test so it would be easier to find it? Thanks